exporter/defoldexporter/exporter.xml

```diff
@@ -88,7 +88,7 @@
 
     <!-- If enabled TexturePacker allows the user to choose non-power-of-2 sizes for sprite
          sheets. Otherwise, only power-of-2 (POT) sizes are enabled in the user interface. -->
-    <supportsNPOT>false</supportsNPOT>
+    <supportsNPOT>true</supportsNPOT>
```

Comment:
Might as well update the settings when updating the version number more officially to 1.1 as it's a bit of a turnaorund for the TP team to integrate the changes.